kong/templates/nginx_kong.lua

```diff
@@ -53,6 +53,7 @@ exit_worker_by_lua_block {
 > if (role == "traditional" or role == "data_plane") and #proxy_listeners > 0 then
 # Load variable indexes
 lua_kong_load_var_index default;
+lua_kong_load_var_index $request_id;
```

Comment:
I was thinking whether this could be combined with the new request id feature but I think it's unlikely since that request id can be controlled by the client and we absolutely want this particular request_id to be unique

Reply:
I think that, now that the kong_request_id is finalized as an immutable ID, we should probably reuse it in this feature, once merged.